src/content/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph.mdx

```diff
@@ -150,106 +149,129 @@ Dashboards
 
     <tr>
       <td>
-Alerts      </td>
+Data partitions
+      </td>
 
       <td>
-[See all alert-related tutorials](/docs/alerts-applied-intelligence/new-relic-alerts/alerts-nerdgraph/nerdgraph-api-nrql-condition-alerts/)      </td>
+[Manage data partitions](/docs/apis/nerdgraph/examples/nerdgraph-data-partition-rules-tutorial)
+      </td>
     </tr>
 
- <tr>
+    <tr>
       <td>
-Applied Intelligence     </td>
+Date retention</td>
 
       <td>
-[View and configure topology](/docs/apis/nerdgraph/examples/topology-nerdgraph-tutorial)   </td>
+  [Manage data retention](/docs/telemetry-data-platform/manage-data/manage-data-retention/#api-examples)
+      </td>
     </tr>
+
     <tr>
       <td>
-Workloads      </td>
+Distributed tracing      
+      </td>
 
       <td>
-[View and configure workloads](/docs/apis/nerdgraph/tutorials/nerdgraph-workloads-api-tutorials)      </td>
+* [Query distributed tracing data](/docs/apis/nerdgraph/examples/nerdgraph-distributed-trace-data-tutorial)
+* [Configure Infinite Tracing](/docs/apis/nerdgraph/examples/configure-infinite-tracing-graphql) 
+      </td>
     </tr>
 
     <tr>
-  <td>
-Service levels      </td>
-   
       <td>
-[Configure and manage service levels](/docs/apis/nerdgraph/tutorials/nerdgraph-slm)      </td>
+Manage data      
+      </td>
+
+      <td>
+* [Convert event data to metric data](/docs/telemetry-data-platform/ingest-manage-data/convert-data-metrics/analyze-monitor-data-trends-metrics)
+* [Drop data](/docs/accounts/accounts/data-management/drop-data-using-nerdgraph)
+      </td>
     </tr>
 
     <tr>
       <td>
-Manage keys      </td>
+Manage keys      
```

Comment:
Similarly, "Key Management" would be more intuitive for someone scanning the list in alphabetical order looking for key related stuff.